README.md

```diff
@@ -29,6 +29,10 @@ The `cics-bundle-maven-plugin` currently supports the following CICS bundleparts
  To use the plugins, make sure that:
  * Maven is installed into your environment, or
  * You use a Java IDE that supports Maven, e.g. Eclipse, IntelliJ, VS Code...
+ The CICS bundle deployment API is supported by the CMCI JVM server that must be set up in a WUI region. To use this make sure that:
+ * You have a CICS region that is at CICS® TS V5.6 open beta or later
+ * This region is configured to be a WUI region for the CICSplex that contains the deployment target region
+ * This WUI region must be configured to use the CMCI JVM server
```

Comment:
This last bullet changes from 'is' to 'must be' - would expect it to be 'This WUI region is...' to fit in with the intro to the bullets.